src/components/form/DisplayTableHashes/index.tsx

```diff
@@ -20,17 +23,26 @@ export const DisplayTableHashes = (props: Props) => {
     return [
       {
         // eslint-disable-next-line react/display-name
-        cell: (row: HashArray) => {
-          return (
-            <Hash
-              externalLink
-              {...(url && { href: `/${url}/${row.hash}` })}
-              onClick={() => {
-                if (typeof callbackOnHistoryPush === 'function') callbackOnHistoryPush()
-              }}
-              value={row.hash}
-            />
-          )
+        cell: ({ hash, title }: HashArray) => {
+          if (title) {
+            return (
+              <FlexRow>
+                {title}
+                <ExternalLink href={getOmenMarketURL(hash)} />
+              </FlexRow>
+            )
+          } else {
```

Comment:
This is the hacky part of the PR, I can't use the hash as a title but at the same time I need the hash to get the url.